cmd/ci-test-mapping/map.go

```diff
@@ -41,10 +41,10 @@ var mapCmd = &cobra.Command{
 			}
 
 			// Create or update schema for mapping table
-			tableManager = bigquery.NewMappingTableManager(context.Background(), bigqueryClient)
-			if err := tableManager.Migrate(); err != nil {
-				log.WithError(err).Fatal("could not migrate mapping table")
-			}
+			//tableManager = bigquery.NewMappingTableManager(context.Background(), bigqueryClient)
```

Comment:
this table (renamed to be `qe_component_mapping`) needs to be created.  The serviceaccount I have lacks permissions to do this.